cluster/addons/dns/skydns-rc.yaml.in

```diff
@@ -73,6 +73,13 @@ spec:
             scheme: HTTP
           initialDelaySeconds: 30
           timeoutSeconds: 5
+        readinessProbe:
+          httpGet:
+            path: /healthz
+            port: 8080
+            scheme: HTTP
+          initialDelaySeconds: 10
```

Comment:
@thockin Yes, it should be 1 second so that pods can start serving requests even sooner. I will update the PR.